docs/GettingStartedDocs/buildandsign.md

```diff
@@ -36,6 +36,10 @@ directly. For example, to see the host linker options when building C++ code wit
 ```bash
 pkg-config opt/openenclave/share/pkgconfig/oehost-g++.pc --libs
 ```
+## Build options for using Open Enclave SDK with a CMake project
+If you have a CMake project and would like to bring in Open Enclave targets, 
+please look at instructions available [here]
+( https://github.com/Microsoft/openenclave/blob/master/cmake/sdk_cmake_targets_readme.md)
```

Comment:
Space after the open paren and the linebreak are causing this link to be rendered correctly.

Reply:
Thanks. I did that.